js/components/portfolio/OperationsComponent.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * CITTADINANZA DIGITALE
+ * io.italia.it
+ *
+ * @flow
+ */
+
+import * as React from 'react'
+import I18n from '../../i18n'
+import {
+  Body,
+  Grid,
+  Icon,
+  Left,
+  List,
+  ListItem,
+  Right,
+  Row,
+  Text
+} from 'native-base'
+import type { NavigationScreenProp, NavigationState } from 'react-navigation'
+import type { CreditCard, Operation } from '../../lib/portfolio/types'
+import PortfolioAPI from '../../lib/portfolio/portfolio-api'
+import { withNavigation } from 'react-navigation'
+import ROUTES from '../../navigation/routes'
+
+type Props = {
+  parent: string,
+  navigation: NavigationScreenProp<NavigationState>,
+  operations: $ReadOnlyArray<Operation>
+}
+
+type State = {
+  data: $ReadOnlyArray<Operation>
+}
+
+/**
+ * Operations' List component
+ */
+class OperationsList extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props)
+    this.state = { data: Array.from(props.operations) }
+  }
+
+  renderDate(operation: Operation): React.Node {
+    const datetime: string = `${operation.date} - ${operation.time}`
+    if (operation.isNew) {
+      return (
+        <Row>
+          <Icon
+            type="FontAwesome"
+            name="circle"
+            active
+            style={{ marginTop: 6, fontSize: 10, color: '#0066CC' }}
+          />
+          <Text note>{datetime}</Text>
+        </Row>
+      )
+    }
+    return (
+      <Row>
+        <Text note>{datetime}</Text>
+      </Row>
+    )
+  }
+
+  getCard(operation: Operation): ?CreditCard {
+    return PortfolioAPI.getCreditCard(operation.cardId)
```

Comment:
is this method synchronous ? if it is not it must be handled with redux-saga

Reply:
This is temporary mocked code waiting for next step in which data will be requested from pago-pa-proxy.